src/components/element-card/element-card.styles.tsx

```diff
@@ -68,3 +85,41 @@ export const ActionButton = styled.button<IActionButtonStyleProps>`
         }
     }}
 `;
+
+export const DroppableWrapper = styled.div<IDroppableWrapperProps>`
+    ${({ isOver }) =>
+        isOver
+            ? css`
+                  border: 1px dashed ${Color.Primary};
+                  border-radius: 0.25rem;
+                  background: ${Color.Accent.Light[5]};
+                  height: 100%;
+                  z-index: -1;
+              `
+            : css`
+                  border: 1px solid transparent;
+              `}
+    position: absolute;
+    flex-direction: column;
+    height: auto;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0.4rem;
+    gap: 0.25rem;
+
+    svg {
+        color: ${Color.Primary};
+        height: 2.08rem;
+        width: 2.08rem;
+    }
+`;
+
+export const DroppableText = styled(Text.Body)`
+    color: ${Color.Primary};
+    text-align: center;
+    font-size: 1rem;
+    font-weight: semi-bold;
```

Comment:
`semi-bold` is not a font-weight, use the `weight` prop to apply the font-weight instead

```
<DroppableText weight={...}>Drop your element here</DroppableText>
```